tests/testthat/helper-aaa.R

```diff
@@ -60,11 +60,17 @@ renv_test_state <- function() {
   opts$repos[opts$repos == "@CRAN@"] <- "https://cloud.r-project.org"
   opts <- opts[csort(names(opts))]
 
+  envvars <- as.list(Sys.getenv())
+  envvars <- envvars[grep("^RENV_DEFAULT_", names(envvars), invert = TRUE)]
```

Comment:
Let me know if you think we should be tracking these env vars, but they didn't seem that important, and frequently changed.

Reply:
This looks fine to me too.